contracts/PetPark.sol

```diff
@@ -1,7 +1,92 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
+/// @title Assignment-1 Smart Contract for Pet Park
+/// @author Srigowri M V
+/// @dev The functions implemented pass the tests provided in /test/PetPark.spec.js
+contract PetPark {
+    //Constants
+    enum AnimalType{ None, Fish, Cat, Dog, Rabbit,Parrot}
+    enum GenderType {Male, Female}
 
+    //Events
+    event Added(AnimalType _animalType, uint count);
+    event Borrowed(AnimalType _animalType);
+    event Returned(AnimalType _animalType);
 
-contract PetPark {
+    //State Variable Declaration
+    address public owner;
+    mapping(AnimalType => uint) public animalCounts;  // Mapping for Animal Type: Count
+    mapping(address => AnimalType) public borrowerAnimal; // Mapping for Borrower Address : Animal Type
```

Comment:
Consider making a struct and clubbing the age, gender and any other information of borrower